src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/changes/benificialowner/psc/OtherBeneficialOwnerPsc.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.changes.benificialowner.psc;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Objects;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.CompanyIdentification;
+
+public class OtherBeneficialOwnerPsc extends Psc {
```

Comment:
Do we need the `registeredOffice` property?

Reply:
Thanks for that this is a good spot 

Looks like `residentialAddress` was changed to `registeredOffice` in the ODP last Thursday.

This might need some thinking about as it means that `residentialAddress` can no longer be in the abstract class 

[ODP before](https://companieshouse.atlassian.net/wiki/pages/viewpage.action?pageId=4144005155&pageVersion=62)

[ODP After](https://companieshouse.atlassian.net/wiki/pages/viewpage.action?pageId=4144005155&pageVersion=63)